docs/application/tizen-studio/common-tools/device-manager.md

```diff
@@ -37,36 +37,46 @@ To use the Device Manager with a specific device, you must connect the device ov
 <a name="connecting"></a>
 ## Connecting Devices
 
-The Device Manager shows both local devices connected to the system and any configured remote devices:
+Device Manager shows both local devices connected to the system and any configured remote devices:
 
 -   Only connected local devices (both physical and virtual devices) are listed. Disconnected devices automatically disappear from the list. Therefore, the **Connection** switch is disabled.
 -   Remote devices are listed even if the connection is lost. You can turn the connection on and off with the **Connection** switch.
 
-**Figure: Connected devices in the Device Manager**
+**Figure: Connected devices in Device Manager**
 
-![Connected devices in the Device Manager](./media/device_manager_connected_devices.png)
+![Connected devices in Device Manager](./media/device_manager_connected_devices.png)
 
 Connecting a device to a computer depends on the device type:
 
 -   Physical device
 
-    When you connect a local physical device with a USB cable, it automatically appears in the Device Manager.
+    When you connect a local physical device with a USB cable, it automatically appears in Device Manager.
 
 - Virtual device
 
-    To connect a virtual device, click the **Emulator Manager** button (![Emulator Manager](./media/device_manager_emulator_manager_icon.png)), select the virtual device, and click **Launch**.
+    To connect a virtual device, click **Emulator Manager** ![Emulator Manager](./media/device_manager_emulator_manager_icon.png), select the virtual device, and click **Launch**.
 
-    When the emulator finishes booting, the device appears in the Device Manager.
+    When the emulator finishes booting, the device appears in Device Manager.
 
 - Remote device
 
-    To connect a remote device:
+    To connect a remote device, follow these steps:
 
-    1.  Launch the Remote Device Manager by clicking the **Remote Device Manager** button (![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png)).
-    2. Click **Scan Devices** to display a list of devices you can connect to.
-    3. Confirm the IP and port of the remote device and change the **Connection** switch to **ON**.
+    1. In the Tizen Studio menu, select **Tools > Device Manager**.
+	2. In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). 
+        
+		![Remote Device Manager](./media/remote_device_manager.png)
+	3. In the **Remote Device Manager** window that appears, you can scan, add, edit, and delete a device. You can click **Scan Devices** ![Scan Device](./media/dm_scan_devices_icon.png) to see the list of available devices. 
```

Comment:
Suggestion.
```suggestion
	3. In the **Remote Device Manager** window that appears, you can scan, add, edit, and delete a device to connect. You can click **Scan Devices** ![Scan Device](./media/dm_scan_devices_icon.png) to see the list of available devices. 
```

Reply:
@mijin85cho 
If we add "to connect" to the sentence as per your suggestion, then the sentence will be incomplete.
To make it complete, we must add "that you want to connect to".

However, as the introductory line for this whole procedure starts with "To connect a remote device, follow these steps:", I think " a device that you want to connect to" is clear for a reader. To make it crisp and concise, can we retain the original sentence,
"In the **Remote Device Manager** window that appears, you can scan, add, edit, and delete a device."?

Please let me know your valuable opinion on the same.

Reply:
@Reni-90 
>To make it crisp and concise, can we retain the original sentence,

If so, we can keep the original sentence. Thanks for your opinion.

Reply:
Thank you.